composites/OnboardingWizard/Step.js

```diff
@@ -258,7 +258,9 @@ class Step extends React.Component {
 			<div className={`${this.props.classPrefix}--step--container`} ref="stepContainer"
 				tabIndex="-1" aria-labelledby="step-title">
 				<h1 id="step-title">{this.props.title}</h1>
-				{ this.getFieldComponents( this.props.fields ) }
+					<div className={`${this.props.classPrefix}-content-container`}>
```

Comment:
Minor:
- indentation: the div is a sibling of h1 and shouldn't have an additional indentation level
- please add spaces within brackets 
```
{ `${ this.props.classPrefix }-content-container` }
```